### DISCUSSION TITLE
Second Screen

### DISCUSSION THREAD

**Discussion by joaochris** [sircharlo/meeting-media-manager#272] [Ideas] [answered]
It would be nice if when pressing the button to hide the Media Window from the second screen (ALT+Z) the Media Window recognizes that a second screen has been connected, and fills the screen after we have connected the video cable to the computer. That way you wouldn't need to close the program and open it again.

**Comment by sircharlo** [comment, marked as answer]
True; I hadn't thought of this scenario. 
With https://github.com/sircharlo/jw-meeting-media-fetcher/commit/7ebe85623fef98ebd73b916b4fbce0a0e081b636, I've moved the screen detection logic to when media presentation mode is started, and when the media window is shown after being previously hidden. 

This means that new screens will get detected every time this button is pressed:
![image](https://user-images.githubusercontent.com/1275666/166238656-3dd02762-0870-475b-ba56-86ab767e39e6.png)

or this button is pressed:
![image](https://user-images.githubusercontent.com/1275666/166248136-6cef4ef2-22e6-4582-af99-fbc427e500c3.png)

**Reply by joaochris** [reply]
Thank you bro! There's only one thing, when I click on this button on your second print, the main JWMMF media screen goes to the second one, and I can't go back.

https://user-images.githubusercontent.com/76891864/166250848-d9269f1e-2e61-4d3a-b18a-636336a619ff.mp4

**Reply by joaochris** [reply]
Another thing. If we have the 2nd screen with the media window on it, when we disconnect the video cable, it goes back to 1 full screen, and we can't move anything 😂

**Reply by sircharlo** [reply]
Lol. I was moving the wrong window 😆 Fixed in the last commit.

**Reply by sircharlo** [reply]
> Another thing. If we have the 2nd screen with the media window on it, when we disconnect the video cable, it goes back to 1 full screen, and we can't move anything 😂

Hit F11 to exit fullscreen mode in that case.

**Reply by joaochris** [reply]
> Lol. I was moving the wrong window 😆 Fixed in the last commit.

Perfect now bro! Thank you!

**Reply by joaochris** [reply]
> > Another thing. If we have the 2nd screen with the media window on it, when we disconnect the video cable, it goes back to 1 full screen, and we can't move anything 😂
> 
> Hit F11 to exit fullscreen mode in that case.

Wonderful!
